openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -182,6 +182,10 @@ jQuery(function () {
 
     $('#wikiselect').on('focus', function(){$(this).select();})
 
+    $('.checkbox-menu').on('click',function(){
+        $(this).find('.dropdown-menu').toggle();
```

Comment:
One thing we still need to do (regardless of if we go with this markup or stick with the checkboxes is to add an `aria-expanded` attribute to the menu buttons & toggle this value (either `true` or `false` when the menu is expanded. 

Also, we need to initialize this `aria-expanded` attribute and a `role="button"` attribute when the page loads if we're going down a route that allows users to use the menus without JavaScript.